openbb_terminal/stocks/stocks_controller.py

```diff
@@ -219,15 +220,24 @@ def call_search(self, other_args: List[str]):
             "-e",
             "--exchange",
             default="",
+            choices=stocks_helper.format_parse_choices(self.exchange),
+            type=str.lower,
+            metavar="exchange",
+            dest="exchange",
+            help="Search by a specific exchange to find stocks matching the criteria",
+        )
+        parser.add_argument(
+            "-ec",
```

Comment:
a single dash should be associated with only a single letter

Reply:
Thank you for the information. Silly me!